translations/ES_eu (international Spanish really)

```diff
@@ -0,0 +1,128 @@
+      faq:
+        - question: "¿Qué es Bitcoin Cash?"
+          answer:
+            - |
+              Bitcoin Cash is dinero electrónico de usuario a usuario a través de Internet. Es completamente descentralizado; no existe ningún banco central y funciona sin requerir confianza en ningún tercero.
+        - question: "¿Bitcoin Cash es diferente de 'Bitcoin'?"
+          answer:
+            - |
+              Sí. Bitcoin Cash es la continuación del proyecto Bitcoin como dinero digital de usuario a usuario. Es una bifurcación de la blockchain de Bitcoin, con reglas de consenso actualizadas que le permiten crecer y escalar.
+        - question: "¿Si poseo Bitcoin, poseo también Bitcoin Cash automáticamente?"
+          answer:
+            - |
+              Todo aquel que poseía Bitcoin en el momento de la creación de Bitcoin Cash, pasó a tener esa misma cantidad en Bitcoin Cash. Esto significa que todo el que poseía Bitcoin en el momento del bloque 478558 (el 1 de agosto de 2017, alrededor de las 13:16 UTC) obtuvo una cantidad de Bitcoin Cash equivalente a la que tenía de Bitcoin en ese momento. Si sus Bitcoins los guarda algún tercero, como por ejemplo una casa de cambio, tendrá que tratar el asunto de sus Bitcoin Cash con ese tercero.
+            - |
+              Todas las transacciones de Bitcoin y Bitcoin Cash posteriores a la bifurcación del 1 de agosto son totalmente independientes las unas de las otras. Por este motivo, toda cantidad de Bitcoin adquirida tras la bifurcación no incluye nada de Bitcoin Cash, y viceversa.
+        - question: "¿Cómo se trata el asunto de la repetición de las transacciones entre la cadena antigua y la nueva?"
+          answer:
+            - |
+              Las transacciones de Bitcoin Cash transactions usan un algoritmo de hashing de firmas y vienen indicadas por la etiqueta SIGHASH_FORKID. Estas firmas no son válidas en la red de Bitcoin antigua. Esto evita que las transacciones de Bitcoin Cash se reproduzcan en la cadena de Bitcoin y viceversa.
+        - question: "¿Por qué fue necesaria una bifurcación (<i>fork</i>) para crear Bitcoin Cash?"
+          answer:
+            - |
+              El código de Bitcoin establecía un límite máximo de datos de 1MB por bloque, el equivalente a 3 transacciones por segundo. Aunque aumentar este límite era algo sencillo, la comunidad no pudo alcanzar un consenso a pesar de debatirlo durante años.
+        - question: "¿El límite de 1MB por bloque causaba problemas a Bitcoin?"
+          answer:
+            - |
+              Sí. En 2017 la capacidad alcanzó el "umbral invisible". Las comisiones aumentaron drásticamente y las transacciones de Bitcoin se volvieron poco fiables, dándose el caso de que algunos usuarios no conseguián que sus transacciones se confirmasen incluso tras días de espera.
```

Comment:
"Invisible wall" felt OK here to me. Why the change to "invisible threshold"? 

`conseguián` -> `conseguían`

Reply:
Pared sounds like a physical object, I don't think it makes much sense in Spanish to do a literal translation here. Umbral refers to an upper limit in the form of a digit, a number.

What do you think about "el tope invisible"? (don't know if tope is used commonly in LATAM Spanish, other option is "máximo invisible")

Reply:
Uhm. I think the original expression in English itself is somewhat clunky: `(...) capacity hit the 'invisible wall'`. I mean this in the sense that the literal translation in Spanish doesn't feel either more or less clunky to me. If we're not gonna do a literal translation, perhaps `la capacidad alcanzó este límite artificial` would do?

Either way, I find the word `umbral` a bit odd there, because a threshold has a very specific meaning when talking about a system, which is definitely different from the concept of a limit.

Reply:
límite artificial is perfect. I'm not sure how to edit the file without creating a 4th commit, lol. Replied to you via email